components/loading-box/package.json

```diff
@@ -19,22 +19,7 @@
     "react": ">=15"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.40",
-    "@govuk-react/button": "^0.1.27",
-    "@govuk-react/header": "^0.1.27",
-    "@govuk-react/hoc": "^0.1.27",
-    "@govuk-react/input-field": "^0.1.27",
-    "@govuk-react/label-text": "^0.1.27",
-    "@govuk-react/phase-banner": "^0.1.27",
-    "@govuk-react/radio": "^0.1.27",
-    "@storybook/addon-actions": "^3.4.1",
-    "@storybook/addon-knobs": "^3.3.15",
-    "@storybook/addons": "^3.4.1",
-    "@storybook/react": "^3.4.1",
-    "enzyme": "^3.3.0",
-    "react-dom": "^16.2.0",
-    "react-transition-group": "^2.2.1",
-    "rimraf": "^2.6.2"
+    "react-transition-group": "^2.2.1"
```

Comment:
is this left in devDep bc its the only component that uses this dependency?

Reply:
yes and it's a peerDependency of the component